src/pdfkit.cs

```diff
@@ -157,11 +601,220 @@ interface PdfAnnotation : NSCoding, NSCopying {
 		[Export ("removeAllAppearanceStreams")]
 		void RemoveAllAppearanceStreams ();
 
+		[NoiOS]
 		[Export ("drawWithBox:")]
 		void Draw (PdfDisplayBox box);
+
+		[NoMac] // [FAIL] Selector not found for PdfKit.PdfAnnotation : [set]action 
+		[Export ("action", ArgumentSemantic.Strong), NullAllowed]
+		PdfAction Action { get; set; }
+
+		[NoMac] // [Mac (10,13)] Headers do not show changes on Mac
+		[Export ("highlighted")]
+		bool Highlighted { [Bind ("isHighlighted")] get; set; }
+
+		[NoMac] // [Mac (10,12)] Headers do not show changes on Mac, but added on iOS 11 headers
```

Comment:
you might want to check Xcode8.x (or xtro) for API decorated with `[mac(10,12)]` as they might have been added by Apple last year (and just not done on XM yet)